README.md

```diff
@@ -321,7 +302,7 @@ This will grab all sObject data from Salesforce and allow you to use this extens
 
 ### Settings
 
-This will open a ForceCode Settings window where you can change all of the options in your force.json file in a GUI!
+This will open a ForceCode Settings window where you can change all of the options in your settings.json file in a GUI!
 
 
 ## Help Develop ForceCode 
```

Comment:
Can we throw an invite link in here? 
https://join.slack.com/t/forcecodeworkspace/shared_invite/enQtNDUxNzc3MzA1NDE0LTRiZmQ0MjdmMjQ0ZmQ0NjM3OTI1MGNkYmY4YjdiNTU3MWEzNGZlZGJjYjIxODA2NTA0YzkwMzhmMGNmZTIzMDc